apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -865,28 +865,20 @@ function handleUncaughtErrors( calypsoPort ) {
 	};
 }
 
-function handleEditorLoaded( calypsoPort ) {
-	const unsubscribe = subscribe( () => {
-		const store = select( 'core/editor' );
+async function handleEditorLoaded( calypsoPort ) {
+	await isEditorReadyWithBlocks();
```

Comment:
`isEditorReadyWithBlocks` is already defined in  https://github.com/Automattic/wp-calypso/blob/0f85e376deba7542da23765e81de04e205414d94/apps/wpcom-block-editor/src/utils.js#L40 and used in this file in a few places, I figured it is better to reuse it instead of reimplementing a similar logic.

I check the numbers in the Performance tab in the dev tools and this still triggers right after the content of the post is rendered, so it should be the same.